jenkins-scripts/dsl/sdformat.dsl

```diff
@@ -1,8 +1,8 @@
 import _configs_.*
 import javaposse.jobdsl.dsl.Job
 
-def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat10', 'sdformat11' ]
-def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'sdformat11', 'main' ]
+def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat10', 'sdformat11', 'sdformat12' ]
+def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'sdformat11', 'sdformat12, 'main' ]
 // nightly and prereleases
 def extra_sdformat_debbuilder = [ 'sdformat12' ]
```

Comment:
I don't know if we're supposed to remove `sdformat12` from `extra_sdformat_debbuilder`  or not.

Reply:
I think it could be removed

Reply:
Ok, removed in https://github.com/ignition-tooling/release-tools/pull/525/commits/ec2431219fb337e5d3428773effb22feafaa4761